declarations/plotly.js/index.d.ts

```diff
@@ -3,8 +3,11 @@ import 'plotly.js'
 declare module 'plotly.js' {
   type Update = { x: Array<number[]>, y: Array<number[]> }
   type ShortLayout = { width: number, height: number }
+  type ShortMarker = {size: number}
+  type ShortData = {x: Array<number[]>, y: Array<number[]>, z: Array<number[]>, mode: string, type: string, marker: ShortMarker, name: string}
```

Comment:
I believe `Array<number[]>` is using two different syntaxes, stick with one and just use `number[][]` 🙂